nifi-nar-bundles/nifi-gcp-bundle/nifi-gcp-processors/src/main/java/org/apache/nifi/processors/gcp/vision/AbstractStartGcpVisionOperation.java

```diff
@@ -24,14 +24,36 @@
 import java.io.InputStream;
 import java.io.InputStreamReader;
 import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
 import org.apache.nifi.annotation.lifecycle.OnStopped;
 import org.apache.nifi.components.PropertyDescriptor;
+import org.apache.nifi.expression.ExpressionLanguageScope;
 import org.apache.nifi.flowfile.FlowFile;
 import org.apache.nifi.processor.ProcessContext;
 import org.apache.nifi.processor.ProcessSession;
 import org.apache.nifi.processor.exception.ProcessException;
+import org.apache.nifi.processor.util.StandardValidators;
 
 public abstract class AbstractStartGcpVisionOperation<B extends com.google.protobuf.GeneratedMessageV3.Builder<B>> extends AbstractGcpVisionProcessor  {
+    public static final PropertyDescriptor FEATURE_TYPE = new PropertyDescriptor.Builder()
+            .name("feature-type")
+            .displayName("Vision Feature Type")
+            .description("Type of GCP Vision Feature")
+            .expressionLanguageSupported(ExpressionLanguageScope.FLOWFILE_ATTRIBUTES)
+            .required(false)
+            .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
+            .defaultValue("TEXT_DETECTION")
+            .build();
+    public static final PropertyDescriptor OUTPUT_BUCKET = new PropertyDescriptor.Builder()
+            .name("output-bucket")
+            .displayName("Output Bucket")
+            .description("Name of the GCS bucket where the output of the Vision job will be persisted.")
```

Comment:
As with the Vision Feature Type property, recommend describing when this property applies.
```suggestion
            .description("Name of the GCS bucket where the output of the Vision job will be persisted. The value of this property applies when the JSON Payload property is configured. The JSON Payload property value can use Expression Language to reference the value of ${output-bucket}")
```